helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
```

Comment:
Thats reasonable, and a good point.  There is no helm here however -- so what about drydock-provisioner or something.  Up to you.